archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -155,7 +155,7 @@ package_lld-git(){
 package_polly-git() {
     pkgdesc="Polly is a high-level loop and data-locality optimizer and optimization infrastructure for LLVM"
     url="https://polly.llvm.org/"
-    depends=("llvm-git=$pkgver-$pkgrel")
+    depends=("llvm-libs-git=$pkgver-$pkgrel" "gcc-libs")
```

Comment:
*I think* gcc-libs is typically not listed as dependency..... Or almost everything would have it...

I guess I'm ok with adding it but then it should at least be consistent.....